encoding/api-basics.html

```diff
@@ -17,8 +17,15 @@
 
 function testDecodeSample(encoding, string, bytes) {
   test(function() {
-    assert_equals(new TextDecoder(encoding).decode(new Uint8Array(bytes)), string);
-    assert_equals(new TextDecoder(encoding).decode(new Uint8Array(bytes).buffer), string);
+    const typedArray = new Uint8Array(bytes);
```

Comment:
This does create a change in that decode now gets passed the same buffer rather than a new one. I guess that's fine though.